cmd/peer-rest-server.go

```diff
@@ -61,225 +61,179 @@ func (s *peerRESTServer) GetLocksHandler(w http.ResponseWriter, r *http.Request)
 	logger.LogIf(ctx, gob.NewEncoder(w).Encode(globalLockServer.DupLockMap()))
 }
 
-// DeletePolicyHandler - deletes a policy on the server.
-func (s *peerRESTServer) DeletePolicyHandler(w http.ResponseWriter, r *http.Request) {
-	if !s.IsValid(w, r) {
-		s.writeErrorResponse(w, errors.New("Invalid request"))
-		return
-	}
+var (
+	deletePolicyHandler      = grid.NewSingleHandler[*grid.MSS, *grid.MSS](grid.HandlerDeletePolicy, grid.NewMSS, grid.NewMSS)
```

Comment:
@klauspost I am interested in a handler that only does Req() but has no Response() elements just err or no error.

I couldn't figure out how to write one - would you be able to guide me? I would name it 

```
grid.NewNotifyHandler[*grid.MSS](grid.HandlerDeletePolicy, grid.NewMSS)
```

?

Reply:
`grid.NoPayload` is made for this:

```
// NoPayload is a type that can be used for handlers that do not use a payload.
type NoPayload struct{}
```